pkg/config/setup/config.go

```diff
@@ -365,6 +366,7 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("secret_backend_skip_checks", false)
 	config.BindEnvAndSetDefault("secret_backend_remove_trailing_line_break", false)
 	config.BindEnvAndSetDefault("secret_refresh_interval", 0)
+	config.SetKnown("secret_audit_file_max_size")
```

Comment:
Let me know if I should set a default and/or bind environment variable

Reply:
Setting a value of `0` works okay in this case. The secretResolver will see the zero and set its own default at configuration time: https://github.com/DataDog/datadog-agent/blob/e0297cc06d2b6d5b40fbaeed97dd6b3669407a6d/comp/core/secrets/secretsimpl/secrets.go#L188.

Reply:
Done !